client/my-sites/domains/domain-management/dns/dns-records-list-item.jsx

```diff
@@ -52,7 +56,7 @@ function DnsRecordsListItem( { type, name, value, actions, disabled, isHeader, r
 					<span>{ value }</span>
 				</div>
 				<div className="dns-records-list-item__data dns-records-list-item__menu">
-					{ ! isHeader && menu }
+					{ ! isHeader && enabledActions > 0 && menu }
```

Comment:
Since we're not going to show the disabled actions at all any longer, what would you think about just filtering them out from the `DndRecordData` component when they are passed in as a prop here like `actions={ actions.filter( ( action ) => ! action.disabled ) }`?

Then instead of adding the extra `enabledActions` variable here, we could use `{ ! isHeader && actions.length > 0 && menu }`

Reply:
One other thought on this...I guess we could go back even further in the call stack and just not return the disabled actions from `getActionsForDnsRecord()` ...then there would be no need to even do the filtering.

Reply:
Thanks for the suggestions! I've update the code, it's way better now.